DEVDOCS-4824 + DEVDOCS-3526: [migrate] Many, remove references to Stoplight + unused docs (#1856)
bigcommerce/dev-docs: CONTRIBUTING.md

- [Making a Quick Edit](#making-a-quick-edit)
- [Editing Locally](#editing-locally)
- [Commit Messages](#commit-messages)
- [Style Guides](#style-guides)
- [Linting &amp; Spell Checking](#linting--spell-checking)
- [Contributing to Other Projects](#contributing-to-other-projects)

Thanks for showing interest in contributing!

The following is a set of guidelines for contributing to BigCommerce's Developer Documentation. These are guidelines, not rules. Use your best judgment, and make proposed changes to this document in a pull request.

## Making a Quick Edit

The easiest way to edit a file is using GitHub's web interface:

> 1. Navigate to the file in github ([making-requests.md](NEED NEW URL), for example)

Reviewer: Line 1 needs the new URL here.

Reviewer 2: added.

- [Making a Quick Edit](#making-a-quick-edit)
- [Editing Locally](#editing-locally)
- [Commit Messages](#commit-messages)
- [Style Guides](#style-guides)
- [Linting &amp; Spell Checking](#linting--spell-checking)
- [Contributing to Other Projects](#contributing-to-other-projects)

Thanks for showing interest in contributing!

The following is a set of guidelines for contributing to BigCommerce's Developer Documentation. These are guidelines, not rules. Use your best judgment, and make proposed changes to this document in a pull request.

## Making a Quick Edit

The easiest way to edit a file is using GitHub's web interface:

> 1. Navigate to the file in github ([making-requests.md](https://github.com/bigcommerce/dev-docs/blob/main/docs/api-docs/getting-started/making-requests.mdx), for example).

Reviewer 2: ```suggestion
1. Navigate to the file in GitHub. For example, [making-requests.md](https://github.com/bigcommerce/dev-docs/blob/main/docs/api-docs/getting-started/making-requests.mdx).

```

2. Click the **pencil** icon to **Edit This File**.
3. Make the edit.
4. Type a commit message.
5. Select **Create a new branch for this commit and start a pull request**.
6. Give the branch a descriptive name.
7. Click **Propose file change**.
8. Give the pull request a descriptive title.
9. Fill in the pull request description.
10. Click **Create pull request**.

That's it! You're done.

## Editing Locally

For more complex changes, fork and edit locally:

> 1. Fork `bigcommerce/dev-docs`

Reviewer 2: ```suggestion
1. Fork `bigcommerce/dev-docs`.

```

> 2. `git clone` the fork to your local machine

Reviewer 2: ```suggestion
2. `git clone` the fork to your local machine.

```

> 3. Make edits in your IDE of choice on a new branch

Reviewer 2: ```suggestion
3. Make edits on a new branch in your IDE of choice.

```

4. Commit and push changes to your remote repo

> 5. Create a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) against `bigcommerce/dev-docs/master`

Reviewer 2: ```suggestion
5. Create a [pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) to `bigcommerce:dev-docs/main`.

```

## Commit Messages

- Always include subject; include body when necessary.
- Use present tense ("Add feature" not "Added feature").
- Use imperative mood ("Fix broken link..." not "Fixes broken link...").
- Separate subject from body with a blank line.
- Limit the subject line to 50 characters.
- Capitalize the subject line.
- Do not end the subject line with a period.
- Use the body to explain what and why versus how.

## Style Guides

- [Documentation Style Guide](_project/_doc_style_guide.md)

> - [API Specification Style Guide](_project/_spec_style_guide.md)

Reviewer 2: (this comment is scope sprawl & should be used to make another ticket) --

I don't think we have this `_project` directory anymore, and the style guides almost certainly aren't current
